conductr-cloudformation.yaml

```diff
@@ -207,6 +274,10 @@ Resources:
         Timeout: 5
         UnhealthyThreshold: 2
         HealthyThreshold: 3
+      Instances:
```

Comment:
It doesn't cause the stack creation to fail, it just means that the registered instances will be out of service initially.

Maybe we should reduce the polling interval?